catalogue/webapp/pages/api/works/[workId].tsx

```diff
@@ -1,14 +1,6 @@
 import { NextApiRequest, NextApiResponse } from 'next';
-import { CatalogueApiError } from '@weco/common/model/catalogue';
-import hasOwnProperty from '@weco/common/utils/has-own-property';
 import { getTogglesFromContext } from '@weco/common/server-data/toggles';
-import { getWork } from 'services/catalogue/works';
-
-export function isCatalogueApiError(
-  response: any
-): response is CatalogueApiError {
-  return Boolean(hasOwnProperty(response, 'type') && response.type === 'Error');
-}
```

Comment:
eslint wasn't letting me commit this because of the explicit 'any', and in this case we already know that every value of `response` has a `type`, so we can just inline the bit we care about.